news/_posts/releases/2020-10-22-prestashop-1-7-7-0-rc-release.md

```diff
@@ -0,0 +1,104 @@
+- [In order detail page, when adding a discount amount with a different currency than the default one, an incorrect value is displayed and an incorrect amount is calculated](https://github.com/PrestaShop/PrestaShop/issues/21323)
+- [An exception is displayed when removing a discount with free gift from an order](https://github.com/PrestaShop/PrestaShop/issues/21438)
+- [Incorrect Product's total when adding a product with a discount gift or changing the Shipping](https://github.com/PrestaShop/PrestaShop/issues/21566)
+- [Incorrect total when updating a quantity's product](https://github.com/PrestaShop/PrestaShop/issues/21659)
+
+**Back Office:**
+- [Ranges are asked while they shouldn't during carrier edition](https://github.com/PrestaShop/PrestaShop/issues/20993)
+- [Following migration, $_GET identifier parameters are missing on "View {..} Page](https://github.com/PrestaShop/PrestaShop/issues/20934)
+- [Features page - an exception is displayed](https://github.com/PrestaShop/PrestaShop/issues/20777)
+- [Stats page - No module has been installed](https://github.com/PrestaShop/PrestaShop/issues/21186)
+- [An error is displayed when disabling or enabling maintenance](https://github.com/PrestaShop/PrestaShop/issues/21430)
+- [Hook displayBackOfficeTop not being called on Login BO Page](https://github.com/PrestaShop/PrestaShop/issues/21379)
+
+
+**Front Office:**
+- [Wrong cart rules discount when using alternative currency](https://github.com/PrestaShop/PrestaShop/issues/20594)
+- [Incorrect total with Free gift + Percent discount cart rule](https://github.com/PrestaShop/PrestaShop/issues/20690)
+- [When we have two currencies enabled - the Currency block and the language block are not working anymore](https://github.com/PrestaShop/PrestaShop/issues/20950)
+
+## Changelog
+
+90 pull requests have been merged since the beta 2 release and 40+ issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. 
+This brings us to more than 1300 merged pull requests for the 1.7.7.0 milestone, it is definitely our biggest yet.
+
+If you are looking for more details about all changes and new features expected in 1.7.7, the [1.7.7 beta version release note](https://build.prestashop.com/news/prestashop-1-7-7-0-beta-release/) is the perfect article for you to read!
+
+
+## Download
+
+You can download PrestaShop 1.7.7 RC1 here:
+
+[Download PrestaShop 1.7.7 RC1 now!](https://www.prestashop.com/en/developers-versions){: .btn .btn-lg .btn-success}
+{: .text-center}
+
+
+## How to help finish the 1.7.7
+
+Your feedback on this RC is essential, the regressions you find right now are as many bugs less on the final version and less patch versions. It also means fewer problems on your (or your customer’s) online store. 
+
+You must:
+
+- Install it and test that there is no regression compared to the 1.7.6 version
+- If you develop modules or themes, test them on this version
+- Test the auto-upgrade plugin
```

Comment:
Maybe not this one since it's only a RC and it block the upgrade to the final release?
Or should the upgrade still be tested in a preprod environment? (maybe this should be detailed then, even though you mentioned it earlier in the article)